cmd/utils.go

```diff
@@ -253,6 +253,7 @@ func getImageMetadata(app, imageName, archivePath string) string {
 					--change 'ENV LANG=en_US.utf8' \
 					--change 'ENV QUAYDIR=/quay-registry' \
 					--change 'ENV QUAYCONF=/quay-registry/conf' \
+					--change 'ENV QUAYRUN=/quay-registry/conf' \
```

Comment:
Why is this line needed?

Reply:
Quay will fail to startup. It's referenced in a systemd template and if it isn't set the templating fails. I believe it was recently introduced.